pyproject.toml

```diff
@@ -27,6 +27,7 @@ requires-python = ">=3.8"
 dependencies = [
     "debugpy>=1.0",
     "ipython>=7.23.1",
+    "comm",
```

Comment:
We should add a `>=0.1` here so it gets picked up by our minimum versions check.

Reply:
Thanks for the review! I will update this